test/evergreen.yml

```diff
@@ -2574,6 +2596,29 @@ buildvariants:
     - name: static-wt-build-test
     - name: format-failure-configs-test
 
+- name: ubuntu1804-asan
```

Comment:
Can you consolidate `compile-asan` and `examples-c-asan-test` into this new variant?

Reply:
The point of this variant is that we don't have a compile-asan. We still need it for the `make-check-asan-test` in the `rhel80` variant. In the follow on ticket I'll create I'll suggest we try and cleanup our evergreen.yml file but for now I'd rather not.

The examples-c-asan-test maybe tricky, I'll try now.

Reply:
Oh I see the examples-c-asan-test is a pull-request task and is now being done as part of the examples-c-test. As such I can just remove it.